app/views/posts/index.html.erb

```diff
@@ -3,9 +3,41 @@
 <head>
   <title>User Posts</title>
 </head>
-<body>
-  <h1>User Posts page</h1>
-  <p>This is a list of posts for user ID xxx.</p>
-  <p>Parameters: <%= params %></p>
-</body>
+  <body>
+    <div class="user-details">
+      <div class="user-header">
+        <img src="<%= @user.photo %>" alt="User Photo" class="user-photo">
+        <div class="user-info">
+          <div class="user-box">
+            <h2><%= @user.name %></h2>
+            <p>Number of Posts: <%= @user.posts_counter %></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="user-posts">
+      <% @posts.each do |post| %>
+        <div class="post-box">
+          <h3><%= post.title %></h3>
```

Comment:
- [ Optional ] I think it would be nice if you make it easy for a user to navigate to the `posts show` page by making the title of the posts clickable. This will make it easy for a user to navigate through different pages on your Blog App.